### PR TITLE
View comments on article

### PR DESCRIPTION
**Pull request by cdesale**
Adds View a list of comments associated with an article

### CODE REVIEW

File: utils/get-comments-by-article-id.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
```
Comment (nc-verity-gregory) [inline utils/get-comments-by-article-id.js:1]: These api calls do not need to be in separate files as you are using export.

**Review by nc-verity-gregory — CHANGES_REQUESTED** [review]
What happens if you view comments on article 38?

This looks good on the whole, but what about articles with no comments? I'm currently stuck in an infinite load.

File: utils/get-comments-by-article-id.js (hunk 1 above)
Comment (cdesale) [reply utils/get-comments-by-article-id.js:1]: I have that folder in main now, I will move them to one file in next commit.

**Review by popatre — APPROVED** [review]
Looks much better with the addition of handling the zero comments

**Pull request merged**